walletlibrary/src/main/java/com/microsoft/did/sdk/credential/service/protectors/IssuanceResponseFormatter.kt

```diff
@@ -0,0 +1,119 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+package com.microsoft.did.sdk.credential.service.protectors
+
+import android.util.Base64
+import com.microsoft.did.sdk.credential.service.*
```

Comment:
diff from the repo shows this was:
```kotlin
import com.microsoft.did.sdk.credential.service.IssuanceResponse
import com.microsoft.did.sdk.credential.service.RequestedAccessTokenMap
import com.microsoft.did.sdk.credential.service.RequestedIdTokenMap
import com.microsoft.did.sdk.credential.service.RequestedSelfAttestedClaimMap
import com.microsoft.did.sdk.credential.service.RequestedVcMap
```
Do we have a style guide that dictates this change?

Reply:
These changes have been automatically made by IDE when I moved the files here. I will check the settings on mine.

Reply:
reverted the styling changes.